Packs/Silverfort/Integrations/Silverfort/Silverfort.py

```diff
@@ -8,84 +10,94 @@
 UPDATE_REQ_RESPONSE = {'result': 'updated successfully!'}
 
 
+def get_jwt_token():
+    app_user_id, app_user_secret = API_KEY.split(":")
+    expire_time_sec = 30
+    # Generate JWT token for authentication to Admin Console
+    current_time = time.time()
+    payload = {
+        "issuer": app_user_id,  # REQUIRED - Generated in the UI
+        "iat": current_time,  # REQUIRED - Issued time - current epoch timestamp
+        "exp": current_time + expire_time_sec  # OPTIONAL - Expire tine - token expiry (default 30 seconds from iat)
+    }
+    return jwt.encode(payload, app_user_secret, algorithm="HS256")
```

Comment:
To make this helper method testable, it would probably be better to have the `app_user_id` and `app_user_secret` as arguments into this method as well as the `current_time` and `expire_time_sec` so the method signature would look something like:

```suggestion
def get_jwt_token(app_user_id: str, app_user_secret: str, current_time: float = time.time(), expire_time_sec: int = 30):
    payload = {
        "issuer": app_user_id,  # REQUIRED - Generated in the UI
        "iat": current_time,  # REQUIRED - Issued time - current epoch timestamp
        "exp": current_time + expire_time_sec  # OPTIONAL - Expire tine - token expiry (default 30 seconds from iat)
    }
    return jwt.encode(payload, app_user_secret, algorithm="HS256")
```

Then move :

```python
API_KEY = demisto.params().get('apikey')
```

Into the `main` method as it's suggested that all integration parameters should be instantiated within the `main` method.